src/golang/cmd/executor/executor/log.go

```diff
@@ -0,0 +1,33 @@
+package executor
+
+import (
+	"github.com/aqueducthq/aqueduct/lib/job"
+	"github.com/dropbox/godropbox/errors"
+	log "github.com/sirupsen/logrus"
+	"os"
+	"path"
+)
+
+var LogsDir = path.Join(os.Getenv("HOME"), ".aqueduct", "server", "logs")
```

Comment:
what's the behavior for enterprise version?

Reply:
As a quick fix maybe we can have a command line flag to specify the redirected log file. This could be empty in enterprise version, in which case we will still print in stdout / stderr